lib/p2p/Pool.ts

```diff
@@ -104,7 +104,10 @@ class Pool extends EventEmitter {
 
     this.logger.info('Connecting to known / previously connected peers');
     await this.nodes.load();
-    await this.connectNodes(this.nodes);
+    const connectPromise = this.connectNodes(this.nodes);
+    connectPromise.then(() => {
+      this.logger.info('Completed start-up connections to known peers.');
+    });
```

Comment:
* no need for `connectPromise`. `then` can be applied to `connectNodes`
* need to add `catch`. it's currently failing tslint

Reply:
Thanks I should've caught this myself, done.